packages/cli/src/databases/entities/ExecutionMetadata.ts

```diff
@@ -11,8 +11,8 @@ export class ExecutionMetadata {
 	})
 	execution: ExecutionEntity;
 
-	@RelationId((executionMetadata: ExecutionMetadata) => executionMetadata.execution)
```

Comment:
This prevented us from using the `executionId` in queries and updates. The relation itself is already defined above.